### DISCUSSION TITLE
Run `npm run prod` when running `php hyde build`?

### DISCUSSION THREAD

**Discussion by ydalton** [hydephp/hyde#256] [Q&A] [answered]
Hi, maybe a dumb question, but I would like to have `php hyde build` build my CSS, since I am using Tailwind. If it's possible, how would I go around doing it?

**Comment by ydalton** [comment, marked as answer]
Welp, I kinda answered this one myself. I should have read the `--help` page, because `php hyde build` has an option `--run-prod` or `--run-dev`. Sorry for the noise! :sweat_smile:

**Reply by emmadesilva** [reply]
Hehe no worries! All commands have a `--help` flag where you can see all options and arguments 

Edit: As a related tip, if you're using Tailwind you can also set `use_play_cdn` to `true` in `config/hyde.php` to automatically compile preview CSS in the browser which can be very convenient when developing frontends

**Reply by ydalton** [reply]
The thing is, the reason I wanted this is so that I could have the HydePHP GitHub actions automatically compile the Tailwind classes on deployment, but I don't see how to do it.

**Reply by emmadesilva** [reply]
Hmm, yeah now that I think of it I'm not sure if there is a way to compile the Tailwind using [`hydephp/action`](https://github.com/hydephp/action), but there should be. I'm gonna look into adding such a feature! What do you think about an option like: `compile_tailwind: true` or similar? (Open for syntax suggestions)

**Reply by ydalton** [reply]
Yeah, `compile_tailwind` seems fine. I have almost zero experience with GitHub Actions though, otherwise I'd like to help!

Edit: actually, it's probably not something specific to Tailwind. Maybe `run_mix`?

**Reply by emmadesilva** [reply]
> Edit: actually, it's probably not something specific to Tailwind. Maybe `run_mix`?

Yeah that's a good call! Or maybe something even more generic like `run_build`?

**Reply by emmadesilva** [reply]
Going to prioritise this. Tracking in https://github.com/hydephp/action/issues/30.

A workaround for now is to have a separate workflow that compiles assets on push.
